src/windowsdesktop/src/sfx/Directory.Build.props

```diff
@@ -37,6 +37,11 @@
     <PackageReference Include="Microsoft.Internal.Runtime.WindowsDesktop.Transport" />
   </ItemGroup>
 
+  <!-- Profile is intentionally undefined as this reference will be added via https://github.com/dotnet/wpf/blob/main/packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props#L107 if conditions are met  -->
```

Comment:
The code linked in this comment is not directly related to this.  That code is only in effect when targeting .NET FRamework, and is so that you don't have to include `<Reference Include="WindowsFormsIntegration"/>` in the project file.

I would update the comment to say that the profile is left undefined so that the reference will only be included when no profile is specified (ie both Windows Forms and WPF are in use - see [here](https://github.com/dotnet/wpf/blob/main/packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props#L45-L46)).

Reply:
Consider making it a permalink so it doesn't shift away:

https://github.com/dotnet/wpf/blob/bbfc24fd13804a191e20064acd599b0a359092df/packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props#L45-L46

Reply:
Updated